Makefile

```diff
@@ -6,7 +6,10 @@ F=RUSTFLAGS="-Dwarnings"
 
 # All default features of binaries excluding mutually exclusive features wasmer_sys & wasmer_wamr
 DEFAULT_FEATURES=slow_tests,build_wasms,sqlite-encrypted,hc_demo_cli/build_demo
-UNSTABLE_FEATURES=chc,unstable-dpki,unstable-sharding,unstable-warrants,unstable-functions,unstable-countersigning,$(DEFAULT_FEATURES)
+UNSTABLE_FEATURES=chc,unstable-sharding,unstable-warrants,unstable-functions,unstable-countersigning,$(DEFAULT_FEATURES)
+
+# TODO - removed `unstable-dpki` from the UNSTABLE_FEATURES list
+#        it's so unstable it can't even pass CI
```

Comment:
Haha, while true, can we also note here that we need to release the types libraries before we can update the DPKI DNA and bring a new version into the repo?

Reply:
Updated: https://github.com/holochain/holochain/pull/4838/commits/0c9fde364da27920eadd37400c214904404fdc8a